backend/src/packages/users/user.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { APIPath } from '#libs/enums/enums.js';
+import {
+  type APIHandlerResponse,
+  BaseController,
+} from '#libs/packages/controller/controller.js';
+import { HTTPCode } from '#libs/packages/http/http.js';
+import { type Logger } from '#libs/packages/logger/logger.js';
+import { type UserService } from '#packages/users/user.service.js';
+
+import { UsersApiPath } from './libs/enums/enums.js';
+
+/**
```

Comment:
Can we move to another approach with swagger?

Reply:
We have two options either leave it as it is or do it in a separate folder like it was on guruhub. The problem with the second solution is that it's much harder to maintain. I just chose the better one, to come up with another idea we need a research here